vip-helpers/vip-stats.php

```diff
@@ -92,13 +92,11 @@ function wpcom_vip_get_post_pageviews( $post_id = null, $num_days = 1, $end_date
 		$args['num_days'] = max( 1, min( 90, absint( $args['num_days'] ) ) );
 
 		$posts = stats_get_csv( 'postviews', $args );
-		$views = $posts[0];
+		$views = $posts[0]['views'];
```

Comment:
```suggestion
		$views = $posts[0]['views'] ?? 0;
```
Maybe null coalesce here just to be extra safe.

Reply:
Guess it can't hurt, though I did test lots of failed scenarios and the key was always there.